checker/src/org/checkerframework/checker/lock/LockAnnotatedTypeFactory.java

```diff
@@ -72,7 +122,9 @@ public LockAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFlow) {
     protected Set<Class<? extends Annotation>> createSupportedTypeQualifiers() {
         return Collections.unmodifiableSet(
                 new HashSet<Class<? extends Annotation>>(
```

Comment:
If you push a change that changes the line of code the comment referencing, then the comment will be collapsed.